cmd/images-back/main.go

```diff
@@ -79,16 +82,20 @@ func main() {
 }
 func (env *Env) uploadHandler() echo.HandlerFunc {
 	return func(c echo.Context) error {
-		imgTitle := c.FormValue("title") //name
-		ID, err := models.InsertImage(env.db, imgTitle)
+		file, err := c.FormFile("file")
 		if err != nil {
-			log.Fatal(err)
+			log.Print(err)
```

Comment:
You should `return` from handler here with some error, because `log.Print()` do not stops execution.